packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.controller.js

```diff
@@ -37,40 +39,56 @@ export default class {
     };
   }
 
+  downloadCertificate() {
+    this.trackDashboard('general_information::download_certificate');
+    this.CucControllerHelper.constructor.downloadContent({
+      fileContent: this.database.certificate.ca,
+      fileName: `ca.pem`,
```

Comment:
`fileName` is a constant, it can be declared as a constant and used

Reply:
Fixed :)